.travis.yml

```diff
@@ -3,9 +3,8 @@ python:
   - "3.6"
   - "3.7"
 env:
-  - DJANGO_VERSION='>=1.10,<1.11'
-  - DJANGO_VERSION='>=1.11,<1.12'
   - DJANGO_VERSION='>=2.2,<2.3'
+  - DJANGO_VERSION='>=2.0,<3.3'
```

Comment:
As @symroe mentioned yesterday, Travis no longer runs, so these changes won't do anything. So perhaps a good time to change this out for CircleCI?

Reply:
Absolutely, will do 👍